### ISSUE TITLE
Upgrade Sticky ads to version 1.0

### ISSUE THREAD

**Issue by jasti** [ampproject/amphtml#6169]
**tl;dr**: Upgrade sticky ads component from v0.1 to v1.0 within the next month.

**Introducing AMP Sticky Ads v1.0**

The amp-sticky-ad component was developed as an extension to help publishers monetize ads on AMP pages in a user-friendly manner. We’ve added some improvements to sticky ads which we are launching as version 1.0 to ensure that existing, customized sticky ad implementations that are using version 0.1 don’t break. If you haven’t customized the sticky ads implementation on the page, you can safely upgrade to v1.0

**What’s changing?**
Based on publisher & user research feedback, we are making 3 changes:

**1. UX change to default close button**
The new close button is integrated in a way that it’s part of the sticky ad container, giving it a better look and feel. Moreover, this same close style is also applied to the [amp-app-banner](https://github.com/ampproject/amphtml/blob/master/extensions/amp-app-banner/amp-app-banner.md) for consistency.

![image](https://cloud.githubusercontent.com/assets/2366341/20253053/b9bd2b9e-a9f5-11e6-8c39-afcd28db702b.png)

**2. Restrictions on transparency of the ad container**
User studies at Google (to be released early next year) have shown that an ad that doesn’t span the entire width of the device is considered bad user experience since it may appear as if it’s blocking primary content. Although the default implementation of v0.1 applied an automatic padding so that the sticky ad was always full width, some publisher implementations changed the padding to transparent. With v1.0 we ensure that the ad always spans the entire width of the mobile device, as shown in the picture above.

**3. Removal of intermediate state**
In order to minimize perceived ad loading latency, especially on slow networks, we have removed the intermediary “ad loading” UX. In v1.0, the user will directly see the ad appear soon after the user scrolls past the first viewport. If you previously customized this state, please update your implementation.

![image](https://cloud.githubusercontent.com/assets/2366341/20253048/ac24fb9c-a9f5-11e6-9fef-ea9a53db57f5.png)

**What should I do next?**

The amp-sticky-ad v1.0 has [launched](https://github.com/ampproject/amphtml/blob/master/extensions/amp-sticky-ad/amp-sticky-ad.md). Simply update your AMP page to use 
```
<script async custom-element="amp-sticky-ad" src="https://cdn.ampproject.org/v0/amp-sticky-ad-1.0.js"></script>
```
instead of 
```
<script async custom-element="amp-sticky-ad" src="https://cdn.ampproject.org/v0/amp-sticky-ad-0.1.js"></script>
```

If you didn’t customize the sticky-ad implementation, you are all set. If you did, make sure the new version works as expected with your changes.
You can see a sample implementation [here](https://ampbyexample.com/components/amp-sticky-ad/).

**What if I don’t upgrade?** 

In order to avoid supporting backwards incompatible changes for too long, we want to ensure that all pages upgrade to the latest version within 1 month of this announcement date. AMP documents that don’t upgrade to the latest version within this time will be automatically upgraded to v1.0 which may cause breaking changes.

**What if I have feedback about the change?**

As always, we welcome your feedback. You can comment on this issue or reach out directly to @jasti for questions or clarifications. 

CC @ampproject/ads @zhouyx

**Comment by jpettitt** [comment]
Nice, new close UX is much better.

**Comment by jasti** [comment]
Deployment process for #5432

**Comment by rudygalfi** [comment]
Just wondering if this needs to remain open or can be closed?

**Comment by zhouyx** [comment]
let's keep this open as a track issue to the deployment process.

**Comment by jasti** [comment]
Quick update - this is meant to go live in 2 weeks for those who haven't upgraded already.

**Comment by zhouyx** [comment]
close by #9042
